sci-geosciences/gnome-maps/gnome-maps-43.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+inherit gnome.org gnome2-utils meson python-any-r1 xdg
```

Comment:
I suspect we can drop python-any-r1, etc, because I see

````diff
-meson.add_install_script(
-  'meson_post_install.py',
-  datadir,
-  bindir,
-  app_id
+gnome.post_install(
+  glib_compile_schemas: true,
+  gtk_update_icon_cache: true,
+  update_desktop_database: true,
 )
````